### PR TITLE
Add product history 

### PR DESCRIPTION
**Pull request by MarioRodrigues10**

### CODE REVIEW

**Review by RuiL1904 — COMMENTED** [review]
Are you sure a `canceled` order should be shown on the product history? I don't think so.

**Review by ruioliveira02 — COMMENTED** [review]
image
This should be prettier. However, if anything, please make sure it is aligned and remove the |> please

File: lib/store_web/live/product_live/show.html.heex [hunk 1]
```diff
@@ -1,110 +1,141 @@
...
   </div>
+  <div>
+    <%= if @current_user && @current_user.role in [:admin] do %> 
+    <h1 class="uppercase font-bold pt-16 pb-8">Product Dashboard</h1>
+      <table class="border-4 w-full">
+        <thead class="border-2">
+          <tr>
+            <th>Order ID</th>
+            <th>Time</th>
+            <th>Costumer Email</th>
```
Comment (ruioliveira02) [inline lib/store_web/live/product_live/show.html.heex:112]:
```suggestion
            <th>Customer Email</th>
```

File: lib/store_web/live/product_live/show.html.heex [hunk 2]
```diff
@@ -1,110 +1,141 @@
...
+            <% end %>
+          <% end %>
+        </tbody>
+        <tfoot>
+          <tr>
+            <td>Total Price: <%= total_quantity(@product) * @product.price %> €</td>
+          </tr>
+          <tr>
+            <td>Total Quantity: <%= total_quantity(@product) %></td>
+          </tr>
```
Comment (ruioliveira02) [inline lib/store_web/live/product_live/show.html.heex:136]:
```suggestion
            <td><b>Total Price:</b> <%= total_quantity(@product) * @product.price %> €</td>
          </tr>
          <tr>
            <td><b>Total Quantity:</b> <%= total_quantity(@product) %></td>
          </tr>
```

File: .tool-versions [hunk 3]
```diff
@@ -1,3 +1,3 @@
-erlang 25.0.3
+erlang 25.1.2
```
Comment (RuiL1904) [inline .tool-versions:1]: Can you remove this? 🙏🏼

**Review by RuiL1904 — APPROVED** [review]

**Pull request merged**